Earthfile

```diff
@@ -36,10 +37,13 @@ code:
     COPY --dir earthfile2llb/antlrhandler earthfile2llb/dedup earthfile2llb/image \
         earthfile2llb/imr earthfile2llb/variables earthfile2llb/*.go earthfile2llb/
     COPY ./earthfile2llb/parser+parser/*.go ./earthfile2llb/parser/
+    COPY ./earthfile2llb/parser+parser/*.go ./earthfile2llb/parser/
+    COPY ./buildkitd/entrypoint.sh ./shell_scripts/.
```

Comment:
I would maybe move this to its own target to avoid recompiling go code on every shell script change.

Reply:
good idea, I just created a `lintscripts` target, how does that look?